src/Request.ts

```diff
@@ -479,6 +479,10 @@ export default class Request {
       const headers = evt.multiValueHeaders || _.mapObject(evt.headers, (v) => { return [ v ]; });
 
       return _.reduce(headers, (memo: StringArrayOfStringsMap, v, k) => {
+         if (isUndefined(v)) {
```

Comment:
The new types makes many of these props possibly `undefined`. Thus the addition of these checks.